src/components/screens/login/login.js

```diff
@@ -126,7 +126,7 @@ class Login extends React.Component {
               </p>
             </div>
 
-            <form onSubmit={e => e.preventDefault()}>
+            <form onSubmit={this.onFormSubmit}>
```

Comment:
Moving it here causes side effects of log in on drop down selection

Reply:
Fixed